.github/workflows/generate_markdown_files.yml

```diff
@@ -13,4 +13,16 @@ jobs:
       - name: Install deps
         run: pnpm i
       - name: generate-docs
-        run: pnpm -r run api-extractor && pnpm -r run generate-docs && pnpm run fmt
\ No newline at end of file
+        run: pnpm run docs
+      - name: Auto Generate Markdown Files
+        id: generate-markdown
+        uses: EndBug/add-and-commit@v9
+        with:
+          message: "Auto generate markdown files and push"
+          add: "*"
```

Comment:
Only add .md files

Reply:
Done! (had to force push to test and see what files got pushed by bot, it would've pushed nothing if nothing changed)